app/services/form_validator.rb

```diff
@@ -60,6 +61,20 @@ def validate_country(form_element, el_name, errors)
     end
   end
 
+  def validate_postal(form_element, el_name, errors)
+    zip = @params[el_name]
+    country = @params.fetch(:country, :US)
+    if country.respond_to? :to_sym
+      country = country.to_sym
+    else
+      # country is likely to be nil, so set it to our default
```

Comment:
Occasionally (found through tests and hands-on tests), `:country` is explicitly set to `nil`, which `fetch` will return, eschewing the default. So, we've got to check for that.